GHzDACs/Cleanup/dac.py

```diff
@@ -847,50 +861,62 @@ def func():
                     clkinv = not clkinv
                     yield self._setPolarity(chan, clkinv)
                     tries += 1
-    
+
             ans = found, clkinv, PHOF, tries, targetFifo
             returnValue(ans)
+
         return self.testMode(func)
-    
+
     def runBIST(self, cmd, shift, dataIn):
-        """Run a BIST on the given SRAM sequence. (DAC only)"""
+        """
+
```

Comment:
Yes, but... the BIST is broken (it doesn't pass on the JT boards). I tried to figure out why and there's very little documentation out there for it, and even less of it is comprehensible, and the BIST code doesn't even follow that, AFAICT.